Tasks/DockerV2/dockerpush.ts

```diff
@@ -79,12 +82,14 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
     let output = "";
     let outputImageName = "";
     let digest = "";
+    let imageSize = "";
     let promise = pushMultipleImages(connection, imageNames, tags, commandArguments, (image, commandOutput) => {
         output += commandOutput;
         outputImageName = image;
-        digest = extractDigestFromOutput(commandOutput);
-        tl.debug("outputImageName: " + outputImageName + "\n" + "commandOutput: " + commandOutput + "\n" + "digest:" + digest);
-        publishToImageMetadataStore(connection, outputImageName, tags, digest, dockerFile).then((result) => {
+        digest = extractFromOutput(commandOutput, matchPatternForDigest);
```

Comment:
Any downside in combining Digest and Size regex into single and extract ?

Reply:
Combined the two regex to one